src/components/Button/Button.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable  react/require-default-props */
```

Comment:
[required] don't disable this rule please, enable it and fix the linters errors. That will help you to master dealing with eslint common errors